hibp/plugin.go

```diff
@@ -16,10 +16,10 @@ func Plugin(ctx context.Context) *plugin.Plugin {
 		},
 		DefaultTransform: transform.FromGo().NullIfZero(),
 		TableMap: map[string]*plugin.Table{
+			"hibp_account":  tableAccount(),
```

Comment:
```suggestion
			"hibp_breached_account":  tableBreachedAccount(),
```

I think breached account is a better table name, it's more descriptive and matches the API more closely.